packages/aws-cdk/README.md

```diff
@@ -746,6 +746,32 @@ NOTICES
 If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 16603".
 ```
 
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ CDK migrate is currently experimental. We make no guarantees about the outcome or stability of the Command or it's output.
+
+Generates a CDK application from an existing Cloudformation template in any of the CDK supported languages. 
+Required Arguments:
+* `--from-path <my_file_path>` - Takes the relative filse path to the JSON or YAML template of your choice
+* `--stack-name <my_stack_name>` - Used to name both the CDK application and stack. 
+Optional Arguments:
```

Comment:
This is supposed to be a title - but will show inline and this will all be one paragraph.